css/styles.css

```diff
@@ -50,15 +52,82 @@ li {
   background-color: #e5e5e5;
 }
 
+.close-menu {
+  color: #fff;
+  font-weight: 600;
```

Comment:
- [ ] Please consider using the right ```font-weight``` and ```font-family``` for the close button, so your design is as close to the Figma design as possible